core/ir/opnd_shared.c

```diff
@@ -744,6 +744,14 @@ opnd_create_base_disp_aarch64(reg_id_t base_reg, reg_id_t index_reg,
         CLIENT_ASSERT(false, "opnd_create_base_disp_aarch64: invalid extend type");
     return opnd;
 }
+
+size_t dcache_zva_size;
```

Comment:
style: Put globals at the top of the file (or after a banner comment).
But: shouldn't this be adjacent to cache_line_size which is in drlibc.c?